### PR TITLE
[daint] Adding RDock

### PR DESCRIPTION
**Pull request by victorusu**
This is user code.

### CODE REVIEW

**Comment by victorusu** [comment]
retest this please

File: jenkins-builds/6.0.UP04-17.08-gpu [hunk 1]
```diff
@@ -46,6 +46,7 @@
  PyExtensions-3.5-CrayGNU-17.08.eb
  QuantumESPRESSO-6.1.0-CrayIntel-17.08.eb           --set-default-module
  R-3.4.2-CrayGNU-17.08.eb                           --set-default-module
+ RDock/RDock-2013.1-CrayGNU-17.08.eb                --set-default-module
```
Comment (gppezzi) [inline jenkins-builds/6.0.UP04-17.08-gpu]: maybe you should remove`RDock/` here? and also for `-mc`?

Comment (victorusu) [reply jenkins-builds/6.0.UP04-17.08-gpu]: 👍

File: easybuild/easyconfigs/r/RDock/RDock-2013.1-CrayGNU-17.08.eb [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+source_urls = ['https://sourceforge.net/projects/rdock/files']
+sources = ['rDock_%(version)s_src.tar.gz']
+
+builddependencies = [
+    ('popt', '1.16'),
+]
+
+prebuildopts  = 'cd build && export LD_LIBRARY_PATH=$LIBRARY_PATH:%(builddir)s/build/linux-g++-64/release/lib && '
+prebuildopts += " perl -i -p -e 's/\/usr\/bin\/g\+\+/CC/g' Makefile && "
```
Comment (gppezzi) [inline easybuild/easyconfigs/r/RDock/RDock-2013.1-CrayGNU-17.08.eb]: 👎  for using `perl` :joy:

Comment (victorusu) [reply easybuild/easyconfigs/r/RDock/RDock-2013.1-CrayGNU-17.08.eb]: 😂

**Comment by victorusu** [comment]
Can any of you merge?

**Pull request merged**